resources/artifact-ignores.properties

```
@@ -408,7 +408,7 @@ kubesphere-extension-0.1.0
 kubesphere-extension-0.2.0
 
 # sshd module was packaged as hpi at least for a time
-sshd
+sshd-1.2
```

What about 1.0 and 1.1?
```suggestion
sshd-1.0
sshd-1.1
sshd-1.2
```